traffic/configure-android.md

```diff
@@ -15,65 +15,57 @@ This article describes how to use Fiddler Everywhere to capture and inspect traf
 To capture and inspect traffic on Android devices, perform the following steps:
 
 1. [Provide the prerequisites](#prerequisites).
-1. [Configure Fiddler Everywhere](#configuring-fiddler-everywhere).
-1. [Configure the Android device](#configuring-the-android-device).
-1. [Inspect the browser traffic](#inspecting-the-browser-traffic).
-1. [Inspect the Android application traffic](#inspecting-the-android-application-traffic).
+1. [Configure Fiddler Everywhere](#configuring-fiddler-everywhere-host).
+1. [Configure the Android device](#configuring-android-devices).
+1. [Inspect the browser traffic](#inspecting-browser-traffic).
+1. [Inspect the Android application traffic](#inspecting-android-application-traffic).
 
 ## Prerequisites
 
-- Install [Fiddler Everywhere v. 1.0.0 and above](https://www.telerik.com/download/fiddler-everywhere).
-- Connect an Android device to the same network or use an Android emulator on the Fiddler Everywhere host machine.
-Ensure that the machine on which Fiddler Everywhere and the Android device run is discoverable on the same network.
+- Install the latest version of [Fiddler Everywhere](https://www.telerik.com/download/fiddler-everywhere).
 
-## Configuring Fiddler Everywhere
+- Connect an Android device to the same network or use an Android emulator on the Fiddler Everywhere host machine. Ensure that the machine on which Fiddler Everywhere and the Android device run is discoverable on the same network.
+
+## Configuring Fiddler Everywhere Host
```

Comment:
We must note that Fiddler's root certificate must be trusted and HTTPS capturing to be enabled on the machine where Fiddler Everywhere is running. We are going to remove this limitation in the future, but we have it for the moment.